### PR TITLE
Core: Use consistent literal definition in TableProperties

### PR DESCRIPTION
**Pull request by aokolnychyi**
This PR unifies the definition style of our constants in `TableProperties`.

### CODE REVIEW

**Comment by aokolnychyi** [comment]
cc @szehon-ho @flyrain @RussellSpitzer @karuppayya @rdblue @kbendick @jackye1995

File: core/src/main/java/org/apache/iceberg/TableProperties.java [hunk 1]
```diff
@@ -69,13 +69,13 @@ private TableProperties() {
   public static final long COMMIT_STATUS_CHECKS_MIN_WAIT_MS_DEFAULT = 1000L; // 1s
 
   public static final String COMMIT_STATUS_CHECKS_MAX_WAIT_MS = "commit.status-check.max-wait-ms";
-  public static final long COMMIT_STATUS_CHECKS_MAX_WAIT_MS_DEFAULT = 60000L; // 1 minute
+  public static final long COMMIT_STATUS_CHECKS_MAX_WAIT_MS_DEFAULT = 60 * 1000L; // 1 minute
```
Comment (RussellSpitzer) [inline core/src/main/java/org/apache/iceberg/TableProperties.java:72]: Not sure why we included the L here, this is the smallest multiplication we do among these literals :) I think we could just remove it now, or maybe we should add L on all the constants if we are afraid of overflows ... Doesn't really matter to me

**Review by RussellSpitzer — APPROVED** [review]

**Review by kbendick — APPROVED** [review]

File: core/src/main/java/org/apache/iceberg/TableProperties.java (hunk 1 above)
Comment (aokolnychyi) [reply core/src/main/java/org/apache/iceberg/TableProperties.java:72]: Agree, let me drop L here and in one more place like this for consistency.

Comment (aokolnychyi) [reply core/src/main/java/org/apache/iceberg/TableProperties.java]: Done.

**Comment by aokolnychyi** [comment]
Thanks, @RussellSpitzer @kbendick!

**Pull request merged**